www/core/nightlies/next_minor_list.xml

```diff
@@ -1,8 +1,10 @@
 <extensionset name="Joomla! Core Nightly Builds" description="Joomla! Core Next Minor Nightly Builds">
-	<extension name="Joomla" element="joomla" type="file" version="5.3.0-alpha4-dev" targetplatformversion="4.4.9" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="5.3.0-alpha4-dev" targetplatformversion="4.4.10" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="5.3.0-alpha4-dev" targetplatformversion="5.0" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="5.3.0-alpha4-dev" targetplatformversion="5.1" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="5.3.0-alpha4-dev" targetplatformversion="5.2" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="5.3.0-alpha4-dev" targetplatformversion="5.3" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="5.3.0-beta2-dev" targetplatformversion="4.4.9" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="5.3.0-beta2-dev" targetplatformversion="4.4.10" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
```

Comment:
```suggestion
```
These 2 lines can be deleted as we require 4.4.11 or 4.4.12(-dev) in the next_minor_extension.xml: https://github.com/joomla/update.joomla.org/blob/383323480856e0ff37610d622fbada1174bbeff0/www/core/nightlies/next_minor_extension.xml#L20